R/parallel.R

```diff
@@ -51,6 +51,9 @@ test_files_parallel <- function(
   # TODO: detect number of CPUs
   num_workers <- min(getOption("Ncpus", 4), length(test_paths), 4)
 
+  message("Starting work pool of ", num_workers,
```

Comment:
Maybe "Starting x processes for parallel testing" ?

Reply:
Yeah. I think if we keep the processes, then we'll need some terminology about them, this is why I started using "work pool", but yeah, it is not great, and we can change it later.

Reply:
@hadley Hmmm, the edges case of "Starting 1 process for parallel testing" is weird. How about just "Starting x test processes"?

Reply:
Actually, when will that occur? Why would we do parallel testing with one process?

Reply:
E.g. when you are starting to write tests for a new package, or more often when you do `filter = "watcher"`. It would be annoying to switch back and forth between parallel and non-parallel modes based on how many files are matching.

Reply:
Oh yeah, makes sense.